ext_tables.sql

```diff
@@ -1,13 +1,14 @@
-#
-# Table structure for table 'tx_kkdownloader_images'
-#
-CREATE TABLE tx_kkdownloader_images (
-	name tinytext,
-	image int(11) unsigned DEFAULT '0',
-	imagepreview int(11) unsigned DEFAULT '0',
-	description text,
-	longdescription text,
-	clicks int(10) DEFAULT '0' NOT NULL,
-	last_downloaded int(11) DEFAULT '0' NOT NULL,
-	ip_last_download varchar(45) DEFAULT '' NOT NULL
-);
+#
+# Table structure for table 'tx_kkdownloader_images'
+#
+CREATE TABLE tx_kkdownloader_images
+(
+	name             tinytext,
```

Comment:
Official indent is "tab" with indent-size 2.
If I keep tab as indent between column and format it will explode the indents on systems with other indent size.
So, either I will keep it or revert this column style format back to 1 space for each parameter like TYPO3 does. What do you prefer?

Reply:
you can keep it